app/models/concerns/orchestration/tftp.rb

```diff
@@ -145,19 +149,19 @@ def queue_tftp_update
     if mac != old.mac
       set_tftp = true
       # clean up old TFTP reservation file
-      if old.tftp?
-        queue.create(:name => _("Remove old TFTP Settings for %s") % old, :priority => 19,
-                     :action => [old, :delTFTP])
-      end
+      queue_tftp_destroy(false, 19) if old.tftp?
```

Comment:
Well, how about to pass `old` as a function param called `host` (defaults to `self`). Let me try that in another update, and only if that does not work I will repeat myself.